app/views/users/_form.html.erb

```diff
@@ -140,9 +140,11 @@
 <% end %>
 
 <% if @user.cached_usergroups.any? %>
-  <script>
+  <% content_for(:javascripts) do %>
```

Comment:
Is this a change we could already merge? I saw a few more `content_for` changes like this and wonder if we can trim down the size of this PR a bit by doing more preparations.

Reply:
https://github.com/theforeman/foreman/pull/9936